frontend/app/views/tier/upgrade/paidToPaid.scala.html

```diff
@@ -0,0 +1,148 @@
+@(
+    currentTier: com.gu.membership.salesforce.Tier,
+    targetTier: com.gu.membership.salesforce.Tier,
+    userFields: model.PrivateFields,
+    pageInfo: model.PageInfo,
+    paidPreviewOpt: Option[model.Zuora.PaidPreview],
+    currentSubscription: model.Zuora.SubscriptionDetails
+)(implicit token: play.filters.csrf.CSRF.Token)
+
+@import helper._
+@import org.joda.time.DateTime
+@import views.support.Prices._
+@import views.support.Dates._
+@import views.support.Asset
+
+@main("Upgrade to " + targetTier, pageInfo=pageInfo) {
+
+    <main role="main" class="page-content l-constrained">
+
+        @fragments.page.pageHeader("Great, glad to see you've decided to become a " + targetTier)
```

Comment:
Fairly sure it's this Who that's glad to see you:

![tumblr_inline_n7ug88zvg11qegzff](https://cloud.githubusercontent.com/assets/123386/6148500/f1fa1512-b1f9-11e4-817b-9de9ce0d71a9.gif)